src/americana.js

```diff
@@ -33,249 +33,21 @@ import * as search from "./search.js";
-    lyrRoad.secondaryExpresswayBridge.casing(),
-    lyrRoad.primaryExpresswayBridge.casing(),
-    lyrRoad.trunkBridge.casing(),
-    lyrRoad.motorwayBridge.casing(),
-
-    lyrRoad.roadSimpleCasingBridge.casing(),
-
-    lyrRoad.tertiaryLinkBridge.fill(),
-    lyrRoad.tertiaryLinkTollBridge.fill(),
-    lyrRoad.secondaryLinkBridge.fill(),
-    lyrRoad.secondaryLinkTollBridge.fill(),
-    lyrRoad.primaryLinkBridge.fill(),
-    lyrRoad.primaryLinkTollBridge.fill(),
-    lyrRoad.roadLinkSimpleFillBridge.fill(),
-    lyrRoad.motorwayLinkBridge.fill(),
-
-    lyrRoad.minorBridge.fill(),
-    lyrRoad.minorTollBridge.fill(),
-    lyrRoad.tertiaryBridge.fill(),
-    lyrRoad.tertiaryTollBridge.fill(),
-    lyrRoad.secondaryBridge.fill(),
-    lyrRoad.secondaryTollBridge.fill(),
-    lyrRoad.primaryBridge.fill(),
-    lyrRoad.primaryTollBridge.fill(),
-    lyrRoad.roadSimpleFillBridge.fill(),
-    lyrRoad.motorwayBridge.fill(),
-
-    lyrRoad.roadBridge.surface(),
-
-    lyrRail.railBridge.dashes(),
-    lyrRail.railServiceBridge.dashes(),
-
-    lyrRail.narrowGaugeBridge.dashes(),
-    lyrRail.narrowGaugeServiceBridge.dashes(),
-
-    lyrRail.lightRailTramBridge.dashes(),
-    lyrRail.lightRailTramServiceBridge.dashes(),
-
-    lyrRail.funicularBridge.dashes(),
-
-    lyrRail.railwayBridge.fill(),
-
-    lyrOneway.bridge,
-    lyrOneway.bridgeLink,
-  ];
-
-  //Render bridge without layer on the lowest bridge layer
-  bridgeLayers.forEach((layer) =>
-    layers.push(
-      Util.filteredClone(layer, ["!", ["has", "layer"]], "_layer_bottom")
-    )
-  );
-
-  //One layer at a time to handle stacked bridges
-  for (let i = 1; i <= 4; i++) {
-    bridgeLayers.forEach((layer) => layers.push(Util.restrictLayer(layer, i)));
-  }
-
-  //If layer is more than 5, just give up and render on a single layer.
-  bridgeLayers.forEach((layer) =>
-    layers.push(
-      Util.filteredClone(
-        layer,
-        [">=", ["coalesce", ["get", "layer"], 0], 5],
-        "_layer_top"
-      )
-    )
-  );
-
-  layers.push(
-    //The labels at the end of the list draw on top of the layers at the beginning.
-    lyrWater.waterwayLabel,
-    lyrWater.waterLabel,
-    lyrWater.waterPointLabel,
-
-    lyrTransportationLabel.bridgeSpacer,
-    lyrTransportationLabel.label,
-
-    lyrPark.label,
-    lyrPark.parkLabel,
-    /* The ref label shows up at lower zoom levels and when the long name doesn't fit */
-    lyrAeroway.airportRefLabel,
-    lyrAeroway.minorAirportRefLabel,
-    lyrAeroway.airportLabel,
-    lyrAeroway.minorAirportLabel,
-    lyrAeroway.airportGate,
-
-    lyrHighwayShield.shield,
-
-    lyrHighwayExit.exits,
-
-    lyrPlace.state,
-    lyrPlace.village,
-    lyrPlace.town,
-    lyrPlace.city,
-    lyrPlace.countryOther,
-    lyrPlace.country3,
-    lyrPlace.country2,
-    lyrPlace.country1,
-    lyrPlace.continent
-  );
-
-  Label.localizeLayers(layers, Label.getLocales());
-
-  return layers;
-}
+import { buildLayers } from "./layer/layers.js";
```

Comment:
I think by convention this would be called index.js, but I’m not sure if that results in anything else happening differently.

Reply:
Done